pkgs/development/python-modules/cooldict/default.nix

```
@@ -0,0 +1,24 @@
+{ ana
+, buildPythonPackage
+, fetchPypi
+, pkgs
```

you should try to avoid importing pkgs, it's a good way to prevent accidental closure bloat.
```suggestion
, lib
```